src/transport/SecureSession.h

```diff
@@ -169,7 +171,20 @@ class SecureSession : public Session
     }
 
     System::Clock::Timestamp GetLastActivityTime() const { return mLastActivityTime; }
+    System::Clock::Timestamp GetLastPeerActivityTime() const { return mLastPeerActivityTime; }
     void MarkActive() { mLastActivityTime = System::SystemClock().GetMonotonicTimestamp(); }
+    void MarkActiveRx() { 
+        mLastPeerActivityTime = System::SystemClock().GetMonotonicTimestamp();
+        MarkActive();
+    }
+
+    bool IsPeerActive() {
+        return ((System::SystemClock().GetMonotonicTimestamp() - GetLastPeerActivityTime()) < kMinActiveTime);
```

Comment:
It will assume the peer is active right after the session is initialized. Initialize `mLastPeerActivityTime` to zero to workaround that?